build.sbt

```diff
@@ -16,30 +16,11 @@
 ThisBuild / organization := "za.co.absa"
 ThisBuild / name         := "spark-partition-sizing"
 
-import Dependencies._
-
 lazy val scala211 = "2.11.12"
 
 ThisBuild / scalaVersion := scala211
 
-lazy val root = (project in file("."))
```

Comment:
I would still ave the root project defined like this. Is there a reason for deletion?

Reply:
It seems like it's small enough, so that config wouldn't be necessary